docs/tasks/debug-application-cluster/audit.md

```diff
@@ -25,36 +27,27 @@ answer the following questions:
     - from where was it initiated?
     - to where was it going?
 
-NOTE: Currently, Kubernetes provides only basic audit capabilities, there is still a lot
-of work going on to provide fully featured auditing capabilities (see [this issue](https://github.com/kubernetes/features/issues/22)).
+## Audit logs
 
-Kubernetes audit is part of [kube-apiserver](/docs/admin/kube-apiserver) logging all requests
-coming to the server. Each audit log contains two entries:
+Kubernetes audit is part of [Kube-apiserver][kube-apiserver] logging all requests
+processed by the server. Each audit log entry contains two entries:
```

Comment:
nit: saying "each entry contains two entries" is confusing. How about `Each audit log entry contains two lines:`?